app/models/bracket.rb

```diff
@@ -44,4 +44,8 @@ def assign_matchups
       matchup.update!(option_a: entry_a, option_b: entry_b)
     end
   end
+
+  def matchup_at(position)
+    return self.matchups.find_by(position: position)
+  end
```

Comment:
This _is_ a nice method. This makes me think that there should be a validation to ensure that there cannot be multiple matchups with the same position. What do you think?